src/components/Link.react.js

```diff
@@ -2,69 +2,72 @@
 
 import PropTypes from 'prop-types';
 
-import React, {Component} from 'react';
+import React, { Component } from 'react';
 
 /*
  * event polyfill for IE
  * https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent
  */
-function CustomEvent (event, params) {
-    params = params || {
-        bubbles: false,
-        cancelable: false,
-        detail: undefined
-    };
-    var evt = document.createEvent('CustomEvent');
-    evt.initCustomEvent(
-        event, params.bubbles, params.cancelable, params.detail);
-    return evt;
+function CustomEvent(event, params) {
+  params = params || {
+    bubbles: false,
+    cancelable: false,
+    detail: undefined
+  };
+  var evt = document.createEvent('CustomEvent');
+  evt.initCustomEvent(event, params.bubbles, params.cancelable, params.detail);
+  return evt;
 }
 CustomEvent.prototype = window.Event.prototype;
 
 export default class Link extends Component {
-    constructor(props) {
-        super(props);
-        this.updateLocation = this.updateLocation.bind(this);
-    }
+  constructor(props) {
+    super(props);
+    this.updateLocation = this.updateLocation.bind(this);
+  }
```

Comment:
We should really enforce style with eslint, but for now our convention has been tabs with 4 spaces (to match the code that we write in our python context)

Reply:
Ah yep, I have Prettier running by default on save in VS Code, so if there's nothing in eslint then it will format it like this I guess!

Reply:
I'll set up prettier and eslint in CI

Reply:
including a prettierrc in the root.